src/index.html

```diff
@@ -17,6 +29,86 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#logo"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          class="logo_image"
+          alt="Moyo Company Logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav_link">
+          <li class="nav_item">
+            <a
+              class="nav_list is-active"
+              href="#Apple"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              class="nav_list"
+              href="#Samsung"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              class="nav_list"
+              href="#Smartphones"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              class="nav_list"
+              href="#LaptopsAndComputers"
+              data-qa="hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              class="nav_list"
+              href="#Gadgets"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              class="nav_list"
+              href="#Tablets"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              class="nav_list"
+              href="#Photo"
+            >
+              Photo
+            </a>
+          </li>
```

Comment:
Remember to add empty lines between multiline sibling blocks of HTML to add some 'air' and simplify reading.